lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -150,40 +170,45 @@ class TraceElements extends Gatherer {
 
     const {largestContentfulPaintEvt, mainThreadEvents} =
       TraceProcessor.computeTraceOfTab(loadData.trace);
-    /** @type {Array<TraceElementData>} */
-    const backendNodeData = [];
 
     const lcpNodeId = TraceElements.getNodeIDFromTraceEvent(largestContentfulPaintEvt);
     const clsNodeData = TraceElements.getTopLayoutShiftElements(mainThreadEvents);
-    if (lcpNodeId) {
-      backendNodeData.push({nodeId: lcpNodeId});
-    }
-    backendNodeData.push(...clsNodeData);
+    const animatedElementData = TraceElements.getAnimatedElements(mainThreadEvents);
 
-    const traceElements = [];
-    for (let i = 0; i < backendNodeData.length; i++) {
-      const backendNodeId = backendNodeData[i].nodeId;
-      const metricName =
-        lcpNodeId === backendNodeId ? 'largest-contentful-paint' : 'cumulative-layout-shift';
-      const objectId = await driver.resolveNodeIdToObjectId(backendNodeId);
-      if (!objectId) continue;
-      const response = await driver.sendCommand('Runtime.callFunctionOn', {
-        objectId,
-        functionDeclaration: `function () {
-          ${setAttributeMarker.toString()};
-          ${pageFunctions.getNodePathString};
-          ${pageFunctions.getNodeSelectorString};
-          ${pageFunctions.getNodeLabelString};
-          ${pageFunctions.getOuterHTMLSnippetString};
-          ${pageFunctions.getBoundingClientRectString};
-          return setAttributeMarker.call(this, '${metricName}');
-        }`,
-        returnByValue: true,
-        awaitPromise: true,
-      });
+    const backendNodeDataMap = new Map([
+      ['largest-contentful-paint', lcpNodeId ? [{nodeId: lcpNodeId}] : []],
+      ['cumulative-layout-shift', clsNodeData],
+      ['CLS/non-composited-animations', animatedElementData],
```

Comment:
Can we just drop the CLS prefix and name it `non-composited-animations`?

Reply:
How do you feel about my proposal in https://github.com/GoogleChrome/lighthouse/pull/11138#discussion_r458332330 @Beytoven ?